tests/integration/test_remoted/test_socket_communication/test_syslog_message_parser.py

```diff
@@ -0,0 +1,150 @@
+'''
+copyright: Copyright (C) 2015-2022, Wazuh Inc.
+           Created by Wazuh, Inc. <info@wazuh.com>.
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: The 'wazuh-remoted' program is the server side daemon that communicates with the agents.
+       Specifically, this test will check if 'wazuh-remoted' can receive syslog messages through
+       the socket.
+
+components:
+    - remoted
+
+suite: socket_communication
+
+targets:
+    - manager
+
+daemons:
+    - wazuh-remoted
+
+os_platform:
+    - linux
+    - windows
+
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - Debian Buster
+    - Red Hat 8
+    - Ubuntu Focal
+    - Ubuntu Bionic
+    - Windows 10
+    - Windows Server 2019
+    - Windows Server 2016
+
+references:
+    - https://documentation.wazuh.com/current/user-manual/reference/daemons/wazuh-remoted.html
+
+tags:
+    - remoted
+'''
+import os
+import re
+import pytest
+from time import sleep
+
+from wazuh_testing import ARCHIVES_JSON_PATH
+from wazuh_testing.tools import file
+from wazuh_testing.tools.configuration import load_configuration_template, get_test_cases_data
+from wazuh_testing.tools.run_simulator import syslog_simulator
+from wazuh_testing.tools.thread_executor import ThreadExecutor
+
+
+pytestmark = [pytest.mark.tier(level=0)]
+
+# Reference paths
+TEST_DATA_PATH = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+CONFIGURATIONS_PATH = os.path.join(TEST_DATA_PATH, 'configuration_template')
+TEST_CASES_PATH = os.path.join(TEST_DATA_PATH, 'test_cases')
+SYSLOG_SIMULATOR_START_TIME = 2
+
+# Configuration and cases data
+t1_configurations_path = os.path.join(CONFIGURATIONS_PATH, 'configuration_syslog_message_parser.yaml')
+t1_cases_path = os.path.join(TEST_CASES_PATH, 'cases_syslog_message_parser.yaml')
+
+# Syslog message IPV6 values test configurations (t1)
+t1_configurations_parameters, t1_configurations_metadata, t1_cases_ids = get_test_cases_data(t1_cases_path)
+t1_configurations = load_configuration_template(t1_configurations_path, t1_configurations_parameters,
+                                                t1_configurations_metadata)
+
+
+@pytest.mark.parametrize('configuration, metadata', zip(t1_configurations, t1_configurations_metadata),
+                         ids=t1_cases_ids)
+def test_syslog_message_ipv6(configuration, metadata, set_wazuh_configuration, truncate_event_logs,
```

Comment:
You are testing ipv4 and ipv6 in test cases, so the name of this test is not valid

Reply:
Done in [here](https://github.com/wazuh/wazuh-qa/pull/3609/commits/b6c0aac937531f132aeb56fd1e5f4d49f589fd39)